cli/asc.js

```diff
@@ -219,7 +219,9 @@ exports.main = function main(argv, options, callback) {
     let transformArgs = args.transform;
     for (let i = 0, k = transformArgs.length; i < k; ++i) {
       let filename = transformArgs[i].trim();
-      if (/\.ts$/.test(filename)) require("ts-node").register({ transpileOnly: true, skipProject: true });
+      if (/\.ts$/.test(filename)) {
+        require("ts-node").register({ transpileOnly: true, skipProject: true, compilerOptions: { target: "ES2016" } });
```

Comment:
There is another ts-node invocation at the start of the file loading the compiler in a development setup, that should probably use the same target.

Reply:
I guess it could be solved in other PR from me: https://github.com/AssemblyScript/assemblyscript/pull/985/files#diff-1d2bb562554675174e389bb738b930cdR223